lib_xua/api/xua_conf_default.h

```diff
@@ -428,13 +428,34 @@
 #define ADAT_TX_INDEX         (0)
 #endif
 
+/* Calculate max ADAT channels based on sample rate range */
```

Comment:
There is a simular set of if/elses around line 459. It would be cleaner if I was something like

sort out MAX_ADAT_CHANS based on SR

set ADAT_TX_MAX_CHANS and ADAT_RX_MAX_CHANS to MAX_ADAT_CHANGES or 0 based on XUA_ADAT_RX/RX_EN